pyfar/plot/_line.py

```diff
@@ -20,14 +20,14 @@ def _time(signal, dB=False, log_prefix=20, log_reference=1, unit=None,
 
     # prepare input
     kwargs = _utils._return_default_colors_rgb(**kwargs)
-    data = signal.time.T
     if dB:
         # avoid any zero-values because they result in -inf in dB data
-        eps = np.finfo(float).eps
-        data = log_prefix * np.log10(np.abs(data) / log_reference + eps)
+        data = dsp.decibel(signal, domain='time', log_prefix=log_prefix,
+                           log_reference=log_reference).T
```

Comment:
Could be shortened without loosing readability :)
```suggestion
        data = dsp.decibel(signal, 'time', log_prefix, log_reference).T
```
Could also be changed below...